src/local-storage.js

```diff
@@ -0,0 +1,13 @@
+export const saveDataLocation = 'myLocalToDo';
```

Comment:
- [ ] Kindly take a step back here because when I reload the page my completed items are not being saved

Reply:
I wanted to ask, is the local storage not being saved at all, or just initially?
Because I just fixed the initial save problem, but otherwise it always saves for me, whenever I make changes to it.

Reply:
As pointed by the previous reviewer,

- The updated array of items is stored in local storage, so the user gets the correct list values after the page reloads.

Reply:
Sorry, I just realized, I hadn't pushed my latest code 😥 .
Can you tell me if it will work now, please?